sources/app/llvm-tablegen/tablegen-parser.dylgram

```diff
@@ -14,19 +14,61 @@ end class;
 
 define variable *tablegen-overrides* = make(<string-table>);
 
+define function save-overrides() => (previous-overrides :: <string-table>)
+  let previous-overrides = *tablegen-overrides*;
+  // FIXME
```

Comment:
By the way, what needs fixing here, does `shallow-copy` not work on tables?

Reply:
It turns out that shallow-copy does work on tables, so I changed the code to use it. Perhaps it did not work in 2010.